### ISSUE TITLE
Problem with changing scheduled date

### ISSUE THREAD

**Issue by milosz-rycko-bozenski** [rubengc/AutomatorWP#6]
Hi,

Sometimes I need to scheduled date for action.

For example, let's get this record from _actionscheduler_actions:

```|     93146 | automatorwp_schedule_actions_execute_action | pending | 2024-10-06 07:39:38 | 2024-10-06 07:39:38  | [32,1961,{"trigger":"woocommerce_anonymous_order_status_product","post_id":712,"order_id":18429,"status":"completed"},1686037184] | O:30:"ActionScheduler_SimpleSchedule":2:{s:22:" * scheduled_timestamp";i:1728200378;s:41:" ActionScheduler_SimpleSchedule timestamp";i:1728200378;} |        0 |        0 | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |        0 | NULL          |```

I need to change date from 2024-10-06 07:39:38 to 2024-10-07 07:39:39, so I modify columns scheduled_date_gmt and scheduled_date_local. Apart from it I change column schedule and update database with serialized data:

```O:30:"ActionScheduler_SimpleSchedule":2:{s:22:" * scheduled_timestamp";i:1728279579;s:41:" ActionScheduler_SimpleSchedule timestamp";i:1728279579;}```

where 1728279579 is timestamp for 2024-10-07 07:39:39.

issue is that in WP Dashboard -> Tools > Action Scheduler in scheduled date column there is current date (every time I refresh page), not scheduled date from database.

So basically my question is what should I do more to change date for this one particular action?

Thanks

**Comment by rubengc** [comment]
Hi @milosz-rycko-bozenski 

Sorry but we can offer support for premium add-ons here, open a support ticket at https://automatorwp.com/contact-us/ and our team will be more than glad to assist you with anything
